docs/vision-usage.rst

```diff
@@ -0,0 +1,190 @@
+Using the Vision API
+====================
+
+Authentication and Configuration
+--------------------------------
+
+- For an overview of authentication in ``gcloud-python``,
+  see :doc:`gcloud-auth`.
+
+- In addition to any authentication configuration, you should also set the
+  :envvar:`GCLOUD_PROJECT` environment variable for the project you'd like
+  to interact with. If you are Google App Engine or Google Compute Engine
+  this will be detected automatically.
+
+- After configuring your environment, create a
+  :class:`Client <gcloud.vision.client.Client>`
+
+  .. doctest::
+
+     >>> from gcloud import vision
+     >>> client = vision.Client()
+
+  or pass in ``credentials`` and ``project`` explicitly
+
+  .. doctest::
+
+     >>> from gcloud import vision
+     >>> client = vision.Client(project='my-project', credentials=creds)
+
+Annotating an Image
+-------------------
+
+Annotate a single image
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. doctest::
+
+    >>> from gcloud import vision
+    >>> client = vision.Client(project="my-project-name")
+    >>> with open('/tmp/car.jpg', 'r') as f:
+    ...     client.annotate(f.read(), vision.FeatureTypes.LABEL_DETECTION, 3)
+
+Annotate multiple images
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. doctest::
+
+     >>> images = (
+     ...     ('./image.jpg', [
+     ...         vision.FeatureTypes.LABEL_DETECTION,
+     ...         vision.FeatureTypes.LANDMARK_DETECTION]),
+     ...     ('./image2.jpg', [
+     ...         vision.FeatureTypes.FACE_DETECTION,
+     ...         vision.FeatureTypes.TEXT_DETECTION]),)
+     >>> annotated_images = []
+     >>> for image, feature_types in images:
+     ...     annotated_images.append(
+     ...         vision_client.annotate(
```

Comment:
The vision api explicitly has a batch interface so that you can batch up multiple `annotate`s into a single request. It'd be preferable to be able to take advantage of this - perhaps by passing the `images` sequence directly to `annotate`